Add Dockerfile

mmacata opened the pull request:
This PR suggests to add a simple Dockerfile building the openeo-web-editor and then copying the build resources into an nginx.
To bulid it, `docker build . -t openeo-web-editor` can be executed, while `docker run -p 8080:80 openeo-web-editor` will start it to test locally at `http://127.0.0.1:8080/`

This could help for easy deployment when no configuration is needed. It could then be connected to an automatic build on dockerhub to always have the latest version.

I think it is plain + general enough to be hosted here at the official repository. What do you think?

m-mohr commented:
Looks good to me, but I'm not a Docker user due to hardware limitations, so can't test.

Two remarks:
1. Could you give a quick intro in the README on how to use it?
2. I don't really get which criteria was used to fill the gitignore file?

mmacata commented:
> Looks good to me, but I'm not a Docker user due to hardware limitations, so can't test.

that's a pity.

> 
> Two remarks:
> 
> 1. Could you give a quick intro in the README on how to use it?

Added it

> 2. I don't really get which criteria was used to fill the gitignore file?

Added this info to the dockerignore file

m-mohr commented:
Thanks.
